### DISCUSSION TITLE
Automating Zotero-Obsidian Integration for Keywords, Abstracts, and Zotero Text Database Updates

### DISCUSSION THREAD

**Discussion by Im0It** [BryanJenksCommunity/FAQ#186] [General]
Hello Bryan, hello everyone,

I am relatively new to this space and know basically nothing about programming/coding. As a PhD student in political science, I want to use Obsidian to organize my literature notes, etc.

Your YouTube videos have opened me up to what is possible with Obsidian, Bryan. They’re really a great resource! I implemented the latest workflow and am curious about a couple of features:

1. How do you automatically add keywords and abstracts to Zotero entries? 
2. You mentioned in a comment below one of your videos that it is possible to automatically update Zotero resources in Obsidian. You referred to GitHub for more information. Could you please point me to the specific repositories or resources where I can find out more?

Thank you for your support and all your wonderful efforts!

Best regards,
Imeh

**Comment by tallguyjenks** [comment]
1. Its part of the automatically extracted metadata of the item, using the template syntax in the paper.md template just captures what was already programmatically provided and drops that content into the output note
2. automatically no, but iteratively yes, as in, you take more notes in zotero and you want to update your note in obsidian on that paper and append new notes or changes, its not robustly fleshed out because its not my workflow but i did get a working proof of concept in several pieces defined but it would be a little work to put together to make it more robust and im not interested in doing that for its own sake at this time

referenced work: its in the issues of the sponsored github repository, unless you are a github sponsor you wont be able to access that code/information
